extensions/amp-form/0.1/test/test-utils.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isFieldDefault} from '../utils';
+
+describes.realWin('amp-form-utils', {}, env => {
+  let doc;
+
+  beforeEach(() => {
+    doc = env.win.document;
+  });
+
+  describe('isFieldDefault', () => {
+    describe('text field', () => {
+      let textField;
+
+      beforeEach(() => {
+        const html = '<input type="text" value="default">';
+        doc.body.insertAdjacentHTML('afterbegin', html);
+        textField = doc.querySelector('input');
+      });
+
+      it("returns true if text field's value matches its default value", () => {
+        textField.value = 'default';
+        expect(isFieldDefault(textField)).to.be.true;
+      });
+
+      it("returns false if text field's value does not match its default value", () => {
+        textField.value = 'not default';
+        expect(isFieldDefault(textField)).to.be.false;
+      });
+    });
+
+    describe('textarea', () => {
+      let textarea;
+
+      beforeEach(() => {
+        const html = '<textarea>default</textarea>';
+        doc.body.insertAdjacentHTML('afterbegin', html);
+        textarea = doc.querySelector('textarea');
+      });
+
+      it("returns true if textarea's value matches its default value", () => {
+        textarea.value = 'default';
+        expect(isFieldDefault(textarea)).to.be.true;
+      });
+
+      it("returns false if textarea's value does not match its default value", () => {
```

Comment:
nit: I think our linter usually catches this, but the convention for the amphtml code base is to use single quotes for

Reply:
There is a single quote in the text `textarea's value`, so I used double quotes in this case. Is this okay?

Reply:
Oh I see now, I guess if the linter allows it then it's fine. Thanks